src/OrdersList/OrdersList.js

```diff
@@ -98,6 +99,12 @@ class OrdersList extends Component {
       records: 'mtypes',
       perRequest: 1000,
     },
+    identifierTypes: {
```

Comment:
could you please leverage this into resources file, so this could be reused? thanks

Reply:
Updated